bin/check_supp_models.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import sys
+import csv
+import argparse
+
+from Fred2.EpitopePrediction import EpitopePredictorFactory
+
+
+def convert_allele_back(allele):
+    if str(allele).startswith("H-2-"):
+        # convert internal Fred2 representation back to the nf-core/epitopeprediction input allele format
+        return allele.replace("H-2-", "H2-")
+    elif allele.startswith("HLA-"):
+        return allele.replace("HLA-", "")
+    else:
+        raise ValueError("Allele type unknown: " + allele + ". Currently expects allele to start either with 'HLA-' or 'H-2-'.")
+
+
+def __main__():
+    parser = argparse.ArgumentParser("Write out information about supported models by Fred2 for installed predictor tool versions.")
```

Comment:
"installed" is misleading here I guess? You mean the tools that are available for this pipeline?